app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::V1::UsersController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
+
+  def get_current_user
+    if current_user
+      render json: User.find(current_user.id)
+    else
+      render json: {
+        "role" => "signed out"
+      }
+    end
+  end
+end
```

Comment:
consider moving this type of authorization logic to a `ParksController` with a `new` action and creating an `authorize_admin` method to call on `before_action` of new